### PR TITLE
To be merged for 7.1 - MTA-4220: Removing --packages option in docs examples

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-4220](https://issues.redhat.com/browse/MTA-4220)

### CODE REVIEW

File: docs/topics/mta-review-reports.adoc [hunk 1]
```diff
@@ -11,6 +11,7 @@ The report examples shown in the following sections are a result of analyzing th
 ifdef::cli-guide[]
 The report was generated using the following command.
 
+// please could i ask for QE input on the following codeblock
 [options="nowrap",subs="attributes+"]
 ----
 $ <{ProductShortName}_HOME>/bin/{mta-cli} --input /home/username/{mta-cli}-source/test-files/jee-example-app-1.0.0.ear/ --output /home/username/{mta-cli}-reports/jee-example-app-1.0.0.ear-report --target eap6 --packages com.acme org.apache
```
Comment (anarnold97) [inline docs/topics/mta-review-reports.adoc:17]:
```suggestion
$ <{ProductShortName}_HOME>/bin/{mta-cli} --input /home/username/{mta-cli}-source/test-files/jee-example-app-1.0.0.ear/ --output /home/username/{mta-cli}-reports/jee-example-app-1.0.0.ear-report --target eap6 
```

File: docs/topics/mta-review-reports.adoc [hunk 2]
```diff
@@ -11,9 +11,10 @@ The report examples shown in the following sections are a result of analyzing th
 ifdef::cli-guide[]
 The report was generated using the following command.
 
+// please could i ask for QE input on the following codeblock
 [options="nowrap",subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/bin/{mta-cli} --input /home/username/{mta-cli}-source/test-files/jee-example-app-1.0.0.ear/ --output /home/username/{mta-cli}-reports/jee-example-app-1.0.0.ear-report --target eap6 --packages com.acme org.apache
+$ <{ProductShortName}_HOME>/bin/{mta-cli} --input /home/username/{mta-cli}-source/test-files/jee-example-app-1.0.0.ear/ --output /home/username/{mta-cli}-reports/jee-example-app-1.0.0.ear-report --target eap6
```
Comment (anarnold97) [inline docs/topics/mta-review-reports.adoc:17]:
```suggestion
$ <{ProductShortName}_HOME>/bin/{mta-cli} --input /home/username/{mta-cli}-source/test-files/jee-example-app-1.0.0.ear/ --output /home/username/{mta-cli}-reports/jee-example-app-1.0.0.ear-report --target eap6
```

File: docs/topics/eclipse-configuring-run.adoc [hunk 3]
```diff
@@ -20,6 +20,7 @@ You can create multiple run configurations. Each run configuration must have a u
 . On the *Input* tab, complete the following fields:
 .. Select a migration path.
 .. Beside the *Projects* field, click *Add* and select one or more projects.
+// please could i ask for QE input on the following bullet point and warning
```
Comment (anarnold97) [inline docs/topics/eclipse-configuring-run.adoc:23]:
```suggestion
// should the following line be deleted?
```

File: docs/topics/review-reports.adoc [hunk 4]
```diff
@@ -10,7 +10,7 @@ The report examples shown in the following sections are a result of analyzing th
 
 ifdef::cli-guide[]
 The report was generated using the following command.
-
+// please could i ask for QE input on the following codeblock
```
Comment (anarnold97) [inline docs/topics/review-reports.adoc:13]:
```suggestion
```

File: docs/topics/review-reports.adoc [hunk 5]
```diff
@@ -10,7 +10,6 @@ The report examples shown in the following sections are a result of analyzing th
 
 ifdef::cli-guide[]
 The report was generated using the following command.
```
Comment (anarnold97) [inline docs/topics/review-reports.adoc:12]:
```suggestion
The report was generated using the following command.

```

File: docs/topics/available-openrewrite-recipes.adoc [hunk 6]
```diff
@@ -14,6 +14,7 @@
 |rewrite.configLocation
 |activeRecipes
 
+// // please could i ask for QE input on the following table row
```
Comment (anarnold97) [inline docs/topics/available-openrewrite-recipes.adoc:17]:
```suggestion
```

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 7]
```diff
@@ -49,7 +49,9 @@ You can analyze more than one application at a time against more than one transf
 
 * Application and internal dependencies only.
 * Application and all dependencies, including known Open Source libraries.
+// please could i ask for QE input on the following line
```
Comment (anarnold97) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:52]:
```suggestion
```

File: docs/topics/yaml-rule-structure-syntax.adoc [hunk 8]
```diff
@@ -136,6 +136,7 @@ To exclude incidents for all such open source dependencies, you can use `--dep-l
 
 `konveyor-analyzer ... --dep-label-selector !konveyor.io/dep-source=open-source`
 
+// please could i ask for QE input on the following lines
```
Comment (anarnold97) [inline docs/topics/yaml-rule-structure-syntax.adoc:139]:
```suggestion
```

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 9]
```diff
@@ -50,6 +50,7 @@ You can analyze more than one application at a time against more than one transf
 * Application and internal dependencies only.
 * Application and all dependencies, including known Open Source libraries.
 * Select the list of packages to be analyzed manually. If you choose this option, type the file name and click *Add*.
+// please could i ask for QE input on the following line
```
Comment (anarnold97) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:53]:
```suggestion
```

**Review by kpunwatk — APPROVED** [review]

File: docs/topics/eclipse-configuring-run.adoc [hunk 10]
```diff
@@ -20,6 +20,7 @@ You can create multiple run configurations. Each run configuration must have a u
 . On the *Input* tab, complete the following fields:
 .. Select a migration path.
 .. Beside the *Projects* field, click *Add* and select one or more projects.
+// should the following line be deleted?
```
Comment (anarnold97) [inline docs/topics/eclipse-configuring-run.adoc:23]:
```suggestion
```

**Review by ibragins — APPROVED** [review]
LGTM

File: docs/topics/eclipse-configuring-run.adoc [hunk 11]
```diff
@@ -20,6 +20,7 @@ You can create multiple run configurations. Each run configuration must have a u
 . On the *Input* tab, complete the following fields:
 .. Select a migration path.
 .. Beside the *Projects* field, click *Add* and select one or more projects.
+// please could i ask for QE input on the following bullet point and warning
 .. Beside the *Packages* field, click *Add* and select one or more the packages.
```
Comment (midays) [inline docs/topics/eclipse-configuring-run.adoc]: packages is removed from UI

**Review by midays — CHANGES_REQUESTED** [review]
1 comment otherwise LGTM

File: docs/topics/eclipse-configuring-run.adoc (hunk 11 above)
Comment (anarnold97) [reply docs/topics/eclipse-configuring-run.adoc]: Thanks - removed in [commit](https://github.com/migtools/mta-documentation/pull/77/commits/c6607b0455dd0997b0e6e150a7839bf23f551740)

**Review by dymurray — APPROVED** [review]

**Pull request merged**